src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/AggregateProcessPPIO.java

```diff
@@ -33,6 +37,10 @@ protected MapperWrapper wrapMapper(MapperWrapper next) {
             };
         };
         xstream.allowTypes(new Class[] { AggregateProcess.Results.class });
+        xstream.omitField(AggregateProcess.Results.class, "aggregateAttribute");
```

Comment:
I will put here some examples so we look both at the same output.

Simple WPS aggregate request without group by attributes:

XML
```
<?xml version="1.0" encoding="UTF-8"?>
<AggregationResults>
	<Max>500.0</Max>
</AggregationResults>
```

JSON
```
{
  "AggregationFunctions": ["Max"],
  "AggregationAttribute": "energy_consumption",
  "GroupByAttributes": [],
  "AggregationResults": [
    [500]
  ]
}
```

WPS request with group by attributes:

XML
```
<?xml version="1.0" encoding="UTF-8"?>
<AggregationResults>
	<GroupByResult>
		<object-array>
			<string>FABRIC</string>
			<big-decimal>500.0</big-decimal>
		</object-array>
		<object-array>
			<string>HOUSE</string>
			<big-decimal>6.0</big-decimal>
		</object-array>
		<object-array>
			<string>SCHOOL</string>
			<big-decimal>60.0</big-decimal>
		</object-array>
	</GroupByResult>
</AggregationResults>
```

JSON
```
{
  "AggregationFunctions": ["Max"],
  "AggregationAttribute": "energy_consumption",
  "GroupByAttributes": ["building_type"],
  "AggregationResults": [
    ["FABRIC",500],
    ["HOUSE",6],
    ["SCHOOL",60]
  ]
}
```